pkg/utils/etcdutil/etcdutil_test.go

```diff
@@ -48,44 +47,11 @@ func TestMain(m *testing.M) {
 
 func TestMemberHelpers(t *testing.T) {
 	re := require.New(t)
-	cfg1 := NewTestSingleConfig(t)
-	etcd1, err := embed.StartEtcd(cfg1)
-	defer func() {
-		etcd1.Close()
-	}()
-	re.NoError(err)
-
-	ep1 := cfg1.LCUrls[0].String()
-	client1, err := clientv3.New(clientv3.Config{
-		Endpoints: []string{ep1},
-	})
-	defer func() {
-		client1.Close()
-	}()
-	re.NoError(err)
-
-	<-etcd1.Server.ReadyNotify()
-
-	// Test ListEtcdMembers
-	listResp1, err := ListEtcdMembers(client1)
-	re.NoError(err)
-	re.Len(listResp1.Members, 1)
-	// types.ID is an alias of uint64.
-	re.Equal(uint64(etcd1.Server.ID()), listResp1.Members[0].ID)
+	servers, client1, clean := NewTestEtcdCluster(t, 2)
```

Comment:
Why change it?

Reply:
NewTestEtcdCluster also checks member num and member id

Reply:
I suggest using add member which is more clear. `NewTestEtcdCluster` looks like an initialization.